tests/test_deployments/test_app.py

```diff
@@ -310,7 +310,7 @@ def setup_class(cls) -> None:
 
     @pytest.mark.skipif(
         platform.system() == "Windows",
-        reason="Needs investigation. Could be: Tendermint process is not killed on teardown in Windows.",
+        reason="The pipe gets filled before the test reaches here, and the node becomes unresponsive.",
```

Comment:
We can make this work by having a different fixture instead of `@wait_for_node_to_run`.

Reply:
Yes please make it work.